docs/site/reference/reserved-binding-keys.md

```diff
@@ -4,11 +4,10 @@ title: 'Reserved binding keys'
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI
 toc_level: 1
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Reserved-binding-keys.html
+permalink: /doc/en/lb4/reference-reserved-binding-keys.html
```

Comment:
Let's capitalize the first letter to be consistent with our urls.

```
 /doc/en/lb4/Reference-reserved-binding-keys.html
```

Reply:
I would like to transition our URL scheme to all-lowercase names and eventually to nested paths (`/reference/reserved-binding-keys.html`) to match the directory layout in `docs/site`.

We are already following the new style in migration guide, see e.g. here:

https://github.com/strongloop/loopback-next/blob/ae6427322451c914ae54f44dbb656981e7fbbb81/docs/site/migration/datasources.md#L7

Reply:
Let's discuss in https://github.com/strongloop/loopback-next/issues/5798